doc/rbd/rbd-mirroring.rst

```diff
@@ -238,19 +287,73 @@ For example::
 .. tip:: You can enable journaling on all new images by default by adding
    ``rbd default features = 125`` to your Ceph configuration file.
 
-Enable Image Mirroring
-----------------------
+Create Image Mirror-Snapshots
+-----------------------------
 
-If the mirroring is configured in ``image`` mode for the image's pool, then it
-is necessary to explicitly enable mirroring for each image within the pool.
-To enable mirroring for a specific image with ``rbd``, specify the
-``mirror image enable`` command along with the pool and image name::
+When using snapshot-based mirroring, mirror-snapshots will need to be created
+whenever it is desired to mirror the changed contents of the RBD image. To
+create a mirror-snapshot manually with ``rbd``, specify the
+``mirror image snapshot`` command along with the pool and image name::
+
+        rbd mirror image snapshot {pool-name}/{image-name}
+
+For example::
+
+        $ rbd --cluster site-a mirror image snapshot image-pool/image-1
+
+By default only ``3`` mirror-snapshots will be created per-image. The most
+recent mirror-snapshot is automatically pruned if the limit is reached prior
+to creating a new mirror-snapshot. The limit can be overridden via the
```

Comment:
nit: Actually we create a new snapshot first and then delete the older ones until the limit is satisfied. Or did you intentionally hide this implementation details? I just think that in your interpretation it might be a bit confusing for a user because sometimes she may see more snapshots that specified in the limit param.

Reply:
I was trying to keep it generic, but I'll just yank that last sentence clause.